### PR TITLE
[python] Add python k8s test images

### PR DESCRIPTION
**Pull request by randomanderson**
Test images for k8s testing were incorrectly in the `dd-trace-py` repo. This moves the image generation into the correct repository.

The mirror PR for dd-trace-py is: https://github.com/DataDog/dd-trace-py/pull/10746

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

File: lib-injection/build/docker/python/dd-lib-python-init-test-django-preinstalled/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM python:3.9
+
+ENV PYTHONUNBUFFERED 1
+ENV DJANGO_SETTINGS_MODULE django_app
+WORKDIR /src
+ADD . /src
+
+# Install ddtrace beforehand to ensure it doesn't get overridden.
+RUN pip install django ddtrace==1.12.0
```
Comment (robertomonteromiguel) [inline lib-injection/build/docker/python/dd-lib-python-init-test-django-preinstalled/Dockerfile:9]: I see this and I think this weblog should not be tested with profiling tests. I also think we should test two things for this app:
- The traces are being sent to the test agent (ok, done).
- The tracer version has not been overwritten by the autoinjection (we don't have this test).

**Review by robertomonteromiguel — CHANGES_REQUESTED** [review]
There are two weblog failing on the CI:
- dd-lib-python-init-test-django-preinstalled: We shouldn't launch the profiling tests with this weblog. We should add more assertion to this weblog: check that we don't override the preinstaled tracer.
- dd-lib-python-init-test-django-gunicorn: https://dd.slack.com/archives/C040E5XBLTF/p1727261792081889

**Comment by robertomonteromiguel** [comment]
Solving these 2 CI failures, the PR is ok

File: lib-injection/build/docker/python/dd-lib-python-init-test-django-preinstalled/Dockerfile (hunk 1 above)
Comment (robertomonteromiguel) [reply lib-injection/build/docker/python/dd-lib-python-init-test-django-preinstalled/Dockerfile:9]: Ok, in this case we should skip only the profiling tests

Comment (randomanderson) [reply lib-injection/build/docker/python/dd-lib-python-init-test-django-preinstalled/Dockerfile:9]: Profiling test skipped and latest workflow run is all green

**Comment by randomanderson** [comment]
Profiling test skipped and latest run is all green

**Review by robertomonteromiguel — APPROVED** [review]

**Pull request merged**